projects/legacy/components/textarea/textarea.style.less

```diff
@@ -167,7 +167,7 @@
     }
 
     &__extra {
-        background-color: var(--tui-status-negative-pale-night);
+        background-color: var(--tui-status-negative-pale);
```

Comment:
Looks like there's a few other `-night` issues, please search through the code and fix them all while you're at it

Reply:
Can I create another PR?

Reply:
Why? It's just removing `-night` in 15 places instead of one

Reply:
fixed